ecs-image-build/Dockerfile

```diff
@@ -7,13 +7,14 @@ ARG APPLICATION_VERSION=""
 ARG AWS_ACCESS_KEY_ID=""
 ARG AWS_SECRET_ACCESS_KEY=""
 ARG AWS_REGION="eu-west-2"
+ARG FILE_NAME=""
 
 WORKDIR /opt
 
 RUN dnf install -y zip gzip tar aws-cli \
     && dnf clean all \
-    && aws s3 cp s3://${S3_RELEASE_BUCKET}/dapperdox.developer.ch.gov.uk/dapperdox.developer.ch.gov.uk-${APPLICATION_VERSION}.zip . \
-    && unzip dapperdox.developer.ch.gov.uk-${APPLICATION_VERSION}.zip \
-    && rm -f dapperdox.developer.ch.gov.uk-${APPLICATION_VERSION}.zip
+    && aws s3 cp s3://${S3_RELEASE_BUCKET}/dapperdox.developer.ch.gov.uk/${FILE_NAME} . \
```

Comment:
You could reference the build argument `APPLICATION_NAME` here to give:

```
&& aws s3 cp s3://${S3_RELEASE_BUCKET}/${APPLICATION_NAME}/${FILE_NAME} . \
```

This potentially could be a templateable Dockerfile